pkg/testing/fixture_install.go

```diff
@@ -699,6 +691,27 @@ func (f *Fixture) collectDiagnostics() {
 	}
 }
 
+// FileNamePrefix returns a sanitized and unique name to be used as prefix for
+// files to be kept as resources for investigation when the test fails.
+func (f *Fixture) FileNamePrefix() string {
+	if f.fileNamePrefix != "" {
+		return f.fileNamePrefix
+	}
+
+	stamp := time.Now().Format(time.RFC3339)
+	// on Windows a filename cannot contain a ':' as this collides with disk
+	// labels (aka. C:\)
+	stamp = strings.ReplaceAll(stamp, ":", "-")
+
+	// Subtest names are separated by "/" characters which are not valid
+	// filenames on Linux.
+	sanitizedTestName := strings.ReplaceAll(f.t.Name(), "/", "-")
+	prefix := fmt.Sprintf("%s-%s", sanitizedTestName, stamp)
```

Comment:
Nit: this could be grouped in a generic utility function `SanitizePath` or whatever other name...